### ISSUE TITLE
How to use ip_basic get dense depth map on waymo dataset

### ISSUE THREAD

**Issue by cheniynan** [TRAILab/CaDDN#62]
Thank for your great work! I'm confuse about how to generate depth map on waymo dataset. First I project lidar point to image plane and get sparse depth map. Then I use[ ip_basic](https://github.com/kujason/monopsr/blob/master/demos/depth_completion/save_lidar_depth_maps.py) to get the dense depth map, but it looks not good, just as follow:
![image](https://user-images.githubusercontent.com/68363091/130360269-b72b0bb2-7694-4c49-8f30-696fad000a2d.png)

**Comment by SirJamie** [comment]
hi, how do you solve this problem?

**Comment by cheniynan** [comment]
> hi, how do you solve this problem?

Hi. Fisrt I convert waymo format data to kitti fromat use mmdetection3d. Then I try to project lidar point into camera plane to get sparse depth map and use ip_basic to get dense depth map later. But I found that didn't work well and I just get the depth map as above.  Then I found that we can get camera-project points directly by the [official dataset tools](https://github.com/waymo-research/waymo-open-dataset/blob/master/docs/quick_start.md). Hope that can help you, but I still comfuse about why I can't get the right sparse depth map by project lidar points into camera plane. Maybe there are some bugs during convert waymo to kitti.
